hw/net/rocker/rocker_world.c

```diff
@@ -97,10 +108,8 @@ enum rocker_world_type world_type(World *world)
 
 const char *world_name(World *world)
 {
-    switch (world->type) {
-    case ROCKER_WORLD_TYPE_OF_DPA:
-        return "OF_DPA";
-    default:
-        return "unknown";
+    if (world) {
```

Comment:
Shouldn't need to check if world is non-null here.  If NULL was passed in, that's a bug in the calling code and I think we'd rather have the device crash (so the bug can get fixed) rather than returning something that sort of look like world is non-NULL.